app/views/reports/_report_chart.html.slim

```diff
@@ -0,0 +1,14 @@
+.row.col-xs-12.col-md-10.offset-md-1.text-xs-center
+  .col-xs-6.col-md-3
+    <b>Year:</b> #{report.year}
+  .col-xs-6.col-md-3
+    <b>Month:</b> #{!report.month.empty? ? report.month : "All"}
```

Comment:
When I do `report.month || "All"` it shows month's number if a month is chosen, but if not, it just shows blank :
![screen shot 2016-11-15 at 2 38 30 pm](https://cloud.githubusercontent.com/assets/19661205/20295676/433db45a-ab41-11e6-9bcf-8e72cf80c240.png)